src/pages/Dashboard/index.jsx

```diff
@@ -1,30 +1,35 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 
-import { Space, Typography } from 'antd';
+import { Button } from 'antd';
 
-import Logo from 'assets/img/logowhite.svg';
-import Dashboard from 'components/Dashboard';
+import TabsBar from 'components/TabsBar';
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports